data/json/items/fake.json

```diff
@@ -0,0 +1,36 @@
+[
+    {
+        "id": "toolset",
+        "type": "TOOL",
+        "symbol": ";",
+        "color": "brown",
+        "name": "integrated toolset",
+        "description": "Dummy item.  If you see this, then something went wrong.",
+        "max_charges": 10000,
+        "use_action": [ "HAMMER", "HACKSAW", "LUMBER", "CROWBAR", "REMOVE_ALL_MODS", { "type": "knife" } ],
+        "qualities": [
+            [ "HAMMER", 3 ], [ "HAMMER_FINE", 1 ],
+            [ "SAW_W", 1 ], [ "SAW_M", 1 ], [ "SAW_M_FINE", 1 ],
+            [ "WRENCH", 1 ], [ "WRENCH_FINE", 1 ],
+            [ "SCREW", 1 ], [ "SCREW_FINE", 1 ],
+            [ "CUT", 2 ], [ "PRY", 1 ], ["DRILL", 1 ],
+            [ "BUTCHER", -100 ]
+        ]
+    },{
+        "id": "fake_goggles",
+        "type": "TOOL",
+        "symbol": ";",
+        "color": "brown",
+        "name": "bionic goggles",
+        "description": "Dummy item.  If you see this, then something went wrong.",
```

Comment:
Consider `"copy-from": "fake_item"` and inherit the `description` and possibly later other flags that are common to fake items